lib/Capabilities.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * @copyright Copyright (c) 2023 Stephan Orbaugh <stephan.orbaugh@nextcloud.com>
+ *
+ * @author Stephan Orbaugh <stephan.orbaugh@nextcloud.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+namespace OCA\Files_DownloadLimit;
+
+use OCP\Capabilities\ICapability;
+
+class Capabilities implements ICapability {
+
+	/**
+	 * @return array{files: array{downloadlimit: bool}}
+	 */
+	public function getCapabilities(): array {
+		return [
+			'files' => [
+				'downloadlimit' => true,
+			]
+		];
```

Comment:
I think I would prefer a `enabled` key, like `end_to_end_encryption` is doing :)